src/server/serveractiveobject.h

```diff
@@ -229,6 +236,7 @@ class ServerActiveObject : public ActiveObject
 
 		If this is set alongside with m_pending_removal, removal takes
 		priority.
+		Note: Do not assign this directly, use markForDeactivation() instead.
 	*/
 	bool m_pending_deactivation = false;
```

Comment:
Same here

Reply:
At least this won't require more dumb getters, nothing is asking for it directly.